samples/README.md

```diff
@@ -0,0 +1,147 @@
+[![Build Status](https://travis-ci.org/IBM/MAX-Image-Caption-Generator.svg?branch=master)](https://travis-ci.org/IBM/MAX-Image-Caption-Generator) [![Website Status](https://img.shields.io/website/http/max-image-caption-generator.max.us-south.containers.appdomain.cloud/swagger.json.svg?label=api+demo)](http://max-image-caption-generator.max.us-south.containers.appdomain.cloud/)
+
+# IBM Developer Model Asset Exchange: Image Caption Generator
+
+This repository contains code to instantiate and deploy an image caption generation model. This model generates captions from a fixed vocabulary that describe the contents of images in the [COCO Dataset](http://cocodataset.org/#home). The model consists of an _encoder_ model - a deep convolutional net using the Inception-v3 architecture trained on [ImageNet-2012 data](http://www.image-net.org/challenges/LSVRC/2012/) - and a _decoder_ model - an LSTM network that is trained conditioned on the encoding from the image _encoder_ model. The input to the model is an image, and the output is a sentence describing the image content.
+
+The model is based on the [Show and Tell Image Caption Generator Model](https://github.com/tensorflow/models/tree/master/research/im2txt). The checkpoint files are hosted on [IBM Cloud Object Storage](http://max-assets.s3.us.cloud-object-storage.appdomain.cloud/image-caption-generator/1.0/assets.tar.gz). The code in this repository deploys the model as a web service in a Docker container. This repository was developed as part of the [IBM Code Model Asset Exchange](https://developer.ibm.com/code/exchanges/models/).
+
+## Model Metadata
+| Domain | Application | Industry  | Framework | Training Data | Input Data Format |
+| ------------- | --------  | -------- | --------- | --------- | -------------- | 
+| Vision | Image Caption Generator | General | TensorFlow | [COCO](http://cocodataset.org/#home) | Images | 
+
+## References
+* _O. Vinyals, A. Toshev, S. Bengio, D. Erhan._, ["Show and Tell: Lessons learned from the 2015 MSCOCO Image Captioning Challenge"](https://arxiv.org/abs/1609.06647), IEEE transactions on Pattern Analysis and Machine Intelligence, 2016.
```

Comment:
It seems like the formal journal issue is 2017: https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=7505636

Reply:
@xuhdev The journal on arxiv says `IEEE Transactions on Pattern Analysis and Machine Intelligence ( Volume: PP, Issue: 99 , July 2016 )`
Which one should we go with?

Reply:
Normally I would go with the formal publication. ArXiv reference is used only when a formal publication is not out yet. This is because, for example, arXiv articles may change without notice and arXiv in general has no legal obligation to keep the articles alive. (Try to find some pre-2000 arXiv articles and you'll see it's not that easy.)

Reply:
Oops, the PDF on arXiv I believe is only a manuscript that hasn't been formally published yet.

Reply:
I see, I'll update the link too then.

Reply:
If you would like to change the link, I recommend using the DOI link: https://doi.org/10.1109/TPAMI.2016.2587640